### ISSUE TITLE
schema-reader Error: Undefined array key "order"

### ISSUE THREAD

**Issue by gpluess** [VemtoOrg/vemto2-issues#30]
**Error Message:**
schema-reader Error: Undefined array key "order"

**Stack:**
```
schema-reader Error: #0 /Users/myuser/path/to/my/project/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(255): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(2, 'Undefined array...', 'phar:///Applica...', 69)
#1 phar:///Applications/Vemto.app/Contents/Resources/static/VMTTL1/classes/TableRepository.php(69): Illuminate\Foundation\Bootstrap\HandleExceptions->Illuminate\Foundation\Bootstrap\{closure}(2, 'Undefined array...', 'phar:///Applica...', 69)
#2 [internal function]: TableRepository->{closure}(Array)
#3 phar:///Applications/Vemto.app/Contents/Resources/static/VMTTL1/classes/TableRepository.php(68): array_map(Object(Closure), Array)
#4 phar:///Applications/Vemto.app/Contents/Resources/static/VMTTL1/classes/TableRepository.php(49): TableRepository->insertColumn('companies', Array)
#5 phar:///Applications/Vemto.app/Contents/Resources/static/VMTTL1/classes/TableRepository.php(36): TableRepository->addTableColumn(Array)
#6 phar:///Applications/Vemto.app/Contents/Resources/static/VMTTL1/classes/TableRepository.php(21): TableRepository->processAddedColumns(Array)
#7 phar:///Applications/Vemto.app/Contents/Resources/static/VMTTL1/index.php(82): TableRepository->buildTablesFromMigrations()
#8 phar:///Applications/Vemto.app/Contents/Resources/static/VMTTL1/common/Vemto.php(107): {closure}()
#9 phar:///Applications/Vemto.app/Contents/Resources/static/VMTTL1/index.php(18): Vemto::execute('schema-reader', Object(Closure))
#10 /Applications/Vemto.app/Contents/Resources/static/VMTTL1(15): require('phar:///Applica...')
#11 {main}
```

---

Vemto 2.0.4
darwin 23.3.0
Locale: en-GB
App version: 2.0.4

**Comment by MCS-Technology** [comment]
I found the solution by myself, your case may be the same as mine:
- have you deleted some creation migrations?
- if yes, the problem comes from there
- if yes, here's how to do it, you have to go to the table that is causing the problem, in the migration tab, copy the migration file that you had to delete, recreate it in database/migrations/ (must be the same as vemto), then do a "Save changes" in the diagram, and go to the "Changed table" , and choose "update last migration" instead of "create new migration"

that's it !

**Comment by TiagoSilvaPereira** [comment]
Fixed on 2.0.7 (it is recommended to use Database Schema Reader mode, but the issue was fixed on Migration mode too)
